elytron/src/main/resources/schema/wildfly-elytron_5_0.xsd

```diff
@@ -1588,6 +1608,20 @@
                 </xs:documentation>
             </xs:annotation>
         </xs:attribute>
+        <xs:attribute name="client-ssl-context" type="xs:string" use="optional">
+            <xs:annotation>
+                <xs:documentation>
+                    A predefined client-ssl-context that will be used to connect to the jwks endpoint specifien in jku token claim. This configuration is mandatory if you want to use remote keys with jku.
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="host-name-verification-policy" type="xs:string" use="optional">
+            <xs:annotation>
+                <xs:documentation>
+                    A policy that defines how host names should be verified when using HTTPS for fetching jwks. Allowed values: "ANY".
```

Comment:
same remark: is "DEFAULT" also allowed?

Note that in that case, the enum should be part of the schema type (see for example https://github.com/wildfly/wildfly/blob/886cfbf493571d725f302c0850fe1345216d9e18/messaging-activemq/src/main/resources/schema/wildfly-messaging-activemq_5_0.xsd#L932-L940)